Hazel/src/Platform/Windows/WindowsWindow.cpp

```diff
@@ -110,6 +110,9 @@ namespace Hazel {
 					data.EventCallback(event);
 					break;
 				}
+
+                default:
```

Comment:
You should be careful adding `default`, while unfortunately these are macros and not enums adding a `default` makes it much harder to identify when a case statement is missing, some compilers and static analyzers will typically warning if a case statement is missing in a switch based on an enum without a default.

Additionally there is some formatting issues here where this is using spaces and not tabs, like the rest of the code (also a few other places in this PR where that occurs)